concourse/pipeline.yml

```diff
@@ -111,6 +122,17 @@ jobs:
           GOVUK_NOTIFY_EMAIL_TEMPLATE_ID: d4fddfce-1dfd-4ff5-ab42-156e6ba3e76b
           GOVUK_NOTIFY_SMS_TEMPLATE_ID: a2166a39-91f5-4370-bbbf-c59276975d49
           NOTIFY_API_KEY: ((notify-api-key-stg))
+        on_failure:
+          put: govuk-coronavirus-services-tech-slack
+          params:
+            channel: '#govuk-corona-services-tech'
+            username: 'Concourse (Shield Vulnerable Service)'
```

Comment:
```suggestion
            username: 'Concourse (Shielded Vulnerable Service)'
```

Reply:
Good spot 👍.  Can't believe I missed this in four places, plus in the screenshot of the actual Slack alert 🤦.